src/sql/workbench/contrib/dashboard/browser/pages/serverDashboardPage.component.ts

```diff
@@ -68,4 +68,32 @@ export class ServerDashboardPage extends DashboardPage implements OnInit {
 			this._cd.detectChanges();
 		});
 	}
+
+	getContentTabs(): TabConfig[] {
+		return [{
+			id: 'databasesTab',
+			provider: anyProviderName,
+			publisher: undefined,
+			title: nls.localize('databasesTabName', "Databases"),
+			container: {
+				'widgets-container': [
+					{
+						name: nls.localize('explorerWidgetTitle', "Search"),
```

Comment:
Why is this being moved in here? Ideally all this config should be contributed from the JSON